src/views/List.jsx

```diff
@@ -1,15 +1,44 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	const [searchInput, setSearchInput] = useState('');
+
 	const listItemsToDisplay = data.map((item) => {
-		return <ListItem key={item.id} name={item.name} />;
+		return item.name?.toLowerCase().includes(searchInput.toLowerCase()) ? (
+			<ListItem key={item.id} name={item.name} />
+		) : null;
 	});
 
+	const resetDisplayList = (e) => {
+		e.preventDefault();
+		setSearchInput('');
+	};
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<form>
+				<label htmlFor="filter">Filter by item name</label>
+				<br />
+				<input
+					type="text"
+					name="filter"
+					id="filter"
+					value={searchInput}
+					placeholder="Name"
```

Comment:
No need to change anything here, but, giving a more meaningful `placeholder` value would make things way easier for your users! If I'm filtering items, I think that it would make more sense to use `"Search your item"`, or `"Item name"`, or something between those lines!

Reply:
This has been updated from placeholder `Name` to `Item name` to be more user-friendly 👍